test/parallel/test-crypto-rsa-dsa.js

```diff
@@ -238,6 +259,35 @@ assert.throws(() => {
 }
 
 
+//
+// Test DSA signing and verification with PKCS#8 private key
+//
+{
+  const input = 'I AM THE WALRUS';
+
+  // DSA signatures vary across runs so there is no static string to verify
+  // against
+  const sign = crypto.createSign('SHA1');
+  sign.update(input);
+  const signature = sign.sign(dsaPkcs8KeyPem, 'hex');
+
+  const verify = crypto.createVerify('SHA1');
+  verify.update(input);
+
+  assert.strictEqual(verify.verify(dsaPubPem, signature, 'hex'), true);
+
+  // Test the legacy 'DSS1' name.
```

Comment:
I'd be okay with dropping this, that's already checked around line 250.

Reply:
In my curious, is DSS1 same as SHA-1? :thinking: 
I found following line in Node.js Project but I couldn't find what happened historically about the relationship between DSS1 and SHA-1 from Google search.
https://github.com/nodejs/node/blob/b4f58c2cafba120de81fb806b593464089b96d1c/src/node_crypto.cc#L4459

Reply:
DSS1 stands for DSA (Digital Signature Algorithm) with SHA-1 as the hash function. It's a long-deprecated (and now removed) openssl synonym from when openssl conflated public key algorithms with their hash functions.

Reply:
I see. Thanks :+1: :+1: :+1: